index.html

```diff
@@ -744,7 +744,96 @@ <h3 id="searchhistory_title" class="modal-title">Find an old command</h3>
       <!-- ends Increase bash history -->
 
     </div><!-- Ends helpful stuff using terminal -->
-
+    <!-- Start Manage mounted drives and image them -->
+    <div class="well">
+     <h4>Manage Mounted Drives and Image Them</h4>
+
+     <!-- Find a drive location with diskutil  -->
+     <span data-toggle="modal" data-target="#diskutillist"><button type="button" class="btn btn-default" data-toggle="tooltip" data-placement="bottom" title="Find a drive location with diskutil (Mac)">Find a drive location with diskutil (Mac)</button></span>
+     <div id="diskutillist" class="modal fade" tabindex="-1" role="dialog" aria-labelledby="diskutillist_title">
+       <div class="modal-dialog modal-lg">
+         <div class="modal-content">
+           <div class="modal-header">
+             <button type="button" class="close" data-dismiss="modal" aria-label="Close"><span aria-hidden="true">&times;</span></button>
+             <h3 id="whitespacefile_title" class="modal-title">Find a drive location with diskutil (Mac)</h3>
```

Comment:
This h3 tag should have a different ID -- as well as on the subsequent lines. This seems to be an error you didn't introduce per se, there are some other places in the codebase that need to have this id updated.